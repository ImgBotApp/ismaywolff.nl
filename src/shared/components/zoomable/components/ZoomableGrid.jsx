import styled from 'styled-components'

const ZoomableGrid = styled.div`
  display: grid;
  grid-gap: var(--grid-gutter) var(--grid-gutter);
  grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr;
`

export default ZoomableGrid
