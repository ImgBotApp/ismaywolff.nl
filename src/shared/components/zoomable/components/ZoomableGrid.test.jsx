/**
 * @jest-environment node
 */

import 'jest-styled-components'
import React from 'react'
import { shallow } from 'enzyme'
import ZoomableGrid from './ZoomableGrid'

describe('<ZoomableGrid />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<ZoomableGrid />)
    expect(wrapper).toMatchStyledComponentsSnapshot()
  })
})
