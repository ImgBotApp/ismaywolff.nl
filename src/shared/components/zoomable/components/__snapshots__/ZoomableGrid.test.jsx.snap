// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ZoomableGrid /> renders correctly 1`] = `
.c0 {
  display: grid;
  grid-gap: var(--grid-gutter) var(--grid-gutter);
  grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr;
}

<div
  className="c0"
/>
`;
